services-camel/src/test/java/edu/unc/lib/dl/services/camel/solrUpdate/CdrEventToSolrUpdateProcessorTest.java

```diff
@@ -114,92 +112,56 @@ public void testUnknownAction() throws Exception {
 
         processor.process(exchange);
 
-        verify(jmsTemplate, never()).send(any(MessageCreator.class));
+        verify(messageSender, never()).sendIndexingOperation(anyString(), any(Collection.class),
+                any(IndexingActionType.class));
     }
 
     @Test
     public void testMoveAction() throws Exception {
         List<PID> subjects = pidList(NUM_TEST_PIDS);
 
-        Document msgDoc = buildMessage(MOVE.toString(), MOVE.getName(), targetPid, subjects);
+        Document msgDoc = buildMessage(CDRActions.MOVE.toString(), CDRActions.MOVE.getName(), targetPid, subjects);
```

Comment:
Just curious about the reason for this change?  I can just imagine myself switching it back at some point in the future without remembering it'd already changed once before.

Reply:
Confusingly, we have CDRActions.MOVE and IndexingActionsType.MOVE. I was finding that I was having trouble keeping these straight while working on the code, and decided greater specificity might help others (and me) avoid confusion in future